system-validation-test/features/step_definitions/rest_steps.rb

```diff
@@ -41,17 +41,51 @@ def update_query(query, key, value)
   # Send the query
   def submit_query(method, url, options)
     resource_uri = URI(url)
-
     case method.upcase
     when 'GET'
       response = HTTParty.get(resource_uri, options)
+    when 'PUT'
+      response = HTTParty.put(url, options)
+    when 'DELETE'
+      response = HTTParty.delete(resource_uri, options)
     else
       raise "#{method} is not supported yet"
     end
 
     response
   end
+
+  def check_provider(provider_name)
+    url = 'http://localhost:3002/providers'
```

Comment:
This will only check locally.

Reply:
Fixed to now use get_url() to get the correct endpoint for the current environment.